data/assets/scene/milkyway/objects/orionnebula/transforms.asset

```diff
@@ -25,3 +25,14 @@ local NebulaPosition = {
 }
 
 assetHelper.registerSceneGraphNodesAndExport(asset, { NebulaPosition })
+
+
+asset.meta = {
+  Name = "Orion Nebula Transforms",
+  Version = "1.0",
+  Description = [[This asset contains the position of the Orion Nebula. Generally you
+  would not  include this asset as other assets would include it.]],
```

Comment:
Maybe, "..would not explicitly include this asset.." ? 

The part about including it is a little confusing, IMO.